ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -77,6 +110,35 @@ function BlockaidBannerAlert({ txData, ...props }) {
   } = securityAlertResponse;
 
   if (resultType === BlockaidResultType.Benign) {
+    if (isLoadingDisplayed) {
+      return (
+        <Box
+          alignItems={AlignItems.center}
+          borderColor={BorderColor.borderDefault}
+          borderRadius={BorderRadius.SM}
+          padding={2}
+          display={Display.Flex}
+        >
+          <Box
+            alignItems={AlignItems.center}
+            backgroundColor={TextColor.textDefault}
+            borderRadius={BorderRadius.full}
+            className="blockaid_banner__icon_wrapper"
+            display={Display.Flex}
+            justifyContent={JustifyContent.center}
+          >
+            <Icon
+              name={IconName.Check}
+              color={IconColor.primaryInverse}
+              size={IconSize.Sm}
+            />
+          </Box>
+          <Text color={TextColor.textDefault} marginLeft={2}>
+            {t('blockaidNoAlerts')}
+          </Text>
+        </Box>
```

Comment:
the styles appear to be off for some transaction types. disclaimer: I did not check all transaction types
 
<img width="612" alt="Screenshot 2024-02-20 at 2 39 47 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/590f881f-de5d-4b01-a5d1-ca8e74df8c81">
<img width="815" alt="Screenshot 2024-02-20 at 2 56 46 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/374cd872-fd42-46c2-bfad-0c05c2bd2342">

Reply:
That is useful finding @digiwand , I just fixed it.